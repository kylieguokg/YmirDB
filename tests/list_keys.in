list keYs
set hello 28
LIST keys 
set heLLo 19
List keys
set abcdefghijklmno
list keys
set a1b2c3d4e5cd192 10
LIST KEYS
set 100 29 3
list keys 
