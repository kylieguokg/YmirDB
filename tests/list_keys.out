> no keys

> ok

> hello

> ok

> heLLo
hello

> ok

> abcdefghijklmno
heLLo
hello

> ok

> a1b2c3d4e5cd192
abcdefghijklmno
heLLo
hello

> invalid key

> a1b2c3d4e5cd192
abcdefghijklmno
heLLo
hello

> bye
